jobs/build.Jenkinsfile

```diff
@@ -362,6 +362,15 @@ lock(resource: "build-${params.STREAM}") {
             }
         }
 
+        // secex specific tests
+        if (shwrapCapture("cosa meta --get-value images.qemu-secex") != "None") {
+            stage("Kola:TestSecex") {
+                // FIXME: Once we have valid HKD for the builder we should add `--qemu-secex-hostkey /path/to/key` switch and enable reboot tests
+                kola(cosaDir: env.WORKSPACE, arch: basearch, skipBasicScenarios: true, skipSecureBoot: true, skipUpgrade: true,
+                     extraArgs: "--qemu-secex --tag secex")
+            }
+        }
+
```

Comment:
This should not be necessary, as `build` only runs `x86`

Reply:
Thx, will drop